freyja_apps/in_memory/Cargo.toml

```diff
@@ -22,5 +22,6 @@ in-memory-mock-digital-twin-adapter = { workspace = true }
 in-memory-mock-mapping-adapter = { workspace = true }
 in-memory-mock-data-adapter = { workspace = true }
 
-[features]
-containerize = []
+# If built for certain architectures, enable the 'vendored-ssl' feature.
+[target.'cfg(any(target_arch = "aarch64", target_arch = "x86_64"))'.dependencies]
+paho-mqtt = { workspace = true, features = ["vendored-ssl"] }
```

Comment:
is this something we can do in the workspace and reference by `workspace = true` so it doesn't need to be copied to all of the application cargo files?

Reply:
It is a good question. I was having issues with building locally with a combination of this and the config.toml including the statically linked library and I had not tested it in the context of always using vendored-ssl. I am open to trying that but I would like it to be in a separate PR since Lauren needs this for her blueprint work

Reply:
This was a workaround of the issue more so than a permanent fix

Reply:
We can look at this for future work. For now this is okay to unblock blueprint work